Agile_process.md

```diff
@@ -0,0 +1,287 @@
+where SE will need sprints to create the initial prototypes.
+
+## Coordination with User Engagement
+
+User Engagement will also be able to create Sprints and Campaigns. There
+will be situations where UE, based on feedback from users or upcoming
+outreach events, will initiate campaigns around tutorials or user
+requested functionality. For example, based on user feedback, UE may
+determine that by adding a new narrative tree algorithm we could capture
+a significant number of new users. They would write up a Campaign and
+submit it to the EC. In another case, UE may determine that they need
+new material for an upcoming workshop, and this would be a Sprint
+requiring effort from developers and scientists working with the UE.
+
+## Coordination with Release Engineering
+
+We expect that most Sprints will create deployable functionality that
+will be immediately deployed on CI and next.kbase.us and will eventually
+move to production in a rapid time-frame (weeks). Some Campaigns may
+have more disruptive changes (e.g., format changes) that require more
+significant deployment; those migration and deployment activities should
+be integrated into the Campaign and include Sprints to address those
+changes.
+
+## Pre-Sprint Planning
+
+After a campaign is selected, the product owner will meet with the
+sprint team to plan the sprint. During this process, the sprint team and
+product owner will generate user stories, discuss what the deliverables
+of each user story entails and select a set of stories to complete
+during the sprint. At this point, each user story will be broken down
+into tasks. The user stories can then be shared with the project and
+individual tasks can be added to JIRA for tracking.
+
+The sprint-planning team will be responsible for evaluating each task in
+the context of KBase wide integration to prevent redundancy, silos and
+inconsistencies.
+
+## Coordination between Sprint Teams and Box Leads
+
+For Sprints that directly impact areas controlled by a box lead, we
+expect that the Sprint team will include the box lead and/or members of
+that box. Once or twice a week, there will be a Sprint Oversight
+meeting. During this meeting the selected representative from each
+Sprint will meet with the Production Lead, LRM and Box Leads. At this
+meeting both direct and indirect impacts on Box Areas will be discussed
+and activity from the Sprints can be reviewed to ensure that design and
+implementation choices align with the broader project principles and
+goals.
+
+## Reporting and Tracking
+
+The priority list will be maintained and publicly visible. As Campaigns
+are selected for a Sprint Cycle and the planning is completed, the team
+will create and assign JIRA tickets to track tasks.
+
+At the end of each Sprint, the teams will create a formal summary and
+present the results of their Retrospective on the Monday Report Out
+Meeting. These results may also include lessons learned for improving
```

Comment:
"This summary should be linked into the original Sprint issue that was created when the Sprint was approved."